bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/RetryOptions.java

```diff
@@ -92,6 +92,22 @@
     private Set<Status.Code> statusToRetryOn = new HashSet<>(DEFAULT_ENABLE_GRPC_RETRIES_SET);
     private boolean allowRetriesWithoutTimestamp;
 
+    public Builder() {
+    }
+
+    public Builder(RetryOptions options) {
+      enableRetries = options.retriesEnabled;
```

Comment:
minor, but the constructor in BulkOptions uses this.<field> for everything. Would be good to make consistent.